modules/swagger-codegen/src/main/java/io/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -106,6 +107,8 @@ protected Object resolvePropertyToExample(String mediaType, Property property, S
             return 1.23f;
         } else if (property instanceof IntegerProperty) {
             return 123;
+        } else if (property.getClass().equals(BaseIntegerProperty.class)) {
```

Comment:
Other generators have similar fix in place already: https://github.com/swagger-api/swagger-codegen/blob/f70569987a8545b89cfdbb85b2edf1ca8c529f4c/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/Qt5CPPGenerator.java#L275-L282